src/Tribe/Tickets_Handler.php

```diff
@@ -1079,6 +1102,32 @@ public function tickets_order_input( $post_id ) {
 	}
 
 	/**
+	 * Sorts tickets according to stored menu_order
+	 *
+	 * @since TBD
+	 *
+	 * @param array $tickets array of ticket objects
+	 *
+	 * @return array - sorted array of ticket objects
+	 */
+	public function sort_tickets_by_menu_order( $tickets ) {
+		// our sort function
+		function sortTicketsByMenuOrder( $a, $b ) {
```

Comment:
This should be snake_cased. 

I think we should make this a protected method. If memory serves this is a global function in PHP. So now that it's declared, anyone, even other plugins can call `sortTicketsByMenuOrder()` outside this class.